.github/workflows/ci.yml

```diff
@@ -1,13 +1,13 @@
 name: nf-core CI
 # This workflow runs the pipeline with the minimal test dataset to check that it completes without any syntax errors
-# TODO: Uncomment the following section to make the CI run periodically.
-# on:
-#   push:
-#     branches:
-#       - dev
-#   pull_request:
-#   release:
-#     types: [published]
+
+on:
+  push:
```

Comment:
Honestly, this is overkill IMO, you'll lose your mind with notifications :) 

The pull-request and release make sense obvi